lib/standards/css-colors.js

```diff
@@ -0,0 +1,147 @@
+// Source: https://www.w3.org/TR/css-color-3/#svg-color
+const cssColors = {
```

Comment:
There are a few packages available from npm that have these listed in them already. Do we want to be responsible for keeping this list up-to-date, or should we offload that responsibility to the maintainers of one of the existing packages?

Reply:
Also, this is out-of-date. For example ["rebeccapurple" has been added](https://drafts.csswg.org/css-color/#valdef-color-rebeccapurple).

Reply:
Yer right, we should use CSS 4 stuff here. As for why not NPM. Partially because if we do this ourselves we get exactly what we want, and it is more secure.

Reply:
Are we going to remember to update this when changes are made to the spec?

Reply:
I'm cued into W3C publications, I do keep an eye out for things like this.